test/e2e/singlecluster/e2e_test.go

```diff
@@ -17,11 +17,14 @@ limitations under the License.
 package e2e
 
 import (
+	"time"
```

Comment:
There was a small conflict in imports because there is no "github.com/google/go-cmp/cmp/cmpopts" imported in main.